src/index.html

```diff
@@ -1,17 +1,153 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
+    <meta name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label>
+          Surname:
+          <input
+            type="text"
+            name="Surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form-field">
+          Name:
+          <input
+            type="text"
+            name="Name"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            name="YearsOld"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
```

Comment:
```suggestion
        <label class="form-field">
          Name:
          <input
            type="text"
            name="Name"
            autocomplete="off"
          >
        </label>
        
        <label class="form-field">
          How old are You?
          <input
            type="number"
            name="YearsOld"
            value="12"
            min="1"
            max="100"
          >
        </label>
```
Add blank line between same level multiline components
everywhere